tmail-backend/tmail-third-party/ai-bot/src/test/java/com/linagora/tmail/mailet/AIBotConfigTest.java

```diff
@@ -50,6 +75,23 @@ public void shouldThrowRuntimeExceptionWhenURLIsWrong() {
                 .hasMessageContaining("Invalid LLM API base URL");
     }
 
+    @Test
+    public void shouldVerifyCorrectConfiguration() throws AddressException, MalformedURLException {
+        //Arrange
```

Comment:
I did not knew that one.

My credo for testing is GIVEN ... WHEN ... THEN

Or in simpler cases WHEN ... THEN